packages.json

```diff
@@ -20058,5 +20058,19 @@
     "description": "SSH, SCP and SFTP client for Nim",
     "license": "MIT",
     "web": "https://github.com/ba0f3/ssh2.nim"
+  },
+  {
+    "name": "xenon",
+    "url": "https://windowsboy111:Chan Ho Long, Cyrus@github.com/windowsboy111/xenon",
+    "method": "git",
+    "tags": [
+      "wxwidgets",
+      "nim",
+      "webview",
+      "browser"
+    ],
+    "description": "an efficient, simple, and elegant browser.",
+    "license": "GPL-3.0",
+    "web": "https://windowsboy111:Chan Ho Long, Cyrus@github.com/windowsboy111/xenon"
```

Comment:
I think you were accidentally exposed your github password

Reply:
oh freax lmao

Reply:
Idk how my password gets there but can you help me to rm it? thx

Reply:
While at it, can we change `web` field to be a simple `https://github.com/windowsboy111/xenon`?